index.js

```diff
@@ -0,0 +1,205 @@
+const express = require("express");
+const morgan = require("morgan");
+const cors = require("cors");
+const helmet = require("helmet");
+const port = 9000;
+const userDb = require("./data/helpers/userDb.js");
```

Comment:
Forgot to import postDb which will cause errors when trying to operate on posts endpoints.